firefox-ios/Client/Frontend/Theme/ThemedTableViewCells/ThemedTableViewCell.swift

```diff
@@ -60,6 +60,7 @@ class ThemedTableViewCell: UITableViewCell, ReusableCell, ThemeApplicable {
         editingAccessoryView = nil
         editingAccessoryType = .none
         imageView?.image = nil
+        contentView.directionalLayoutMargins = .zero
```

Comment:
Do we need to adjust this margin?

Reply:
Yes because during cell reuse the old margin if applied will be still there and the cell gets layed out with the margin previously applied in a different cell

Reply:
Yes i see that it makes some ui glitches. i'm trying to find a different solution